### ISSUE TITLE
[a11y]: The SVG element has no accessible name in carbon charts

### ISSUE THREAD

**Issue by keithcha** [carbon-design-system/carbon#17342]
### Package

carbon-components, carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

10.58.15, 7.59.24

### React version

17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker 3.1.75

### Description

All charts seem are failing "The SVG element has no accessible name".  The one I'm interested in is the Donut chart, but they all seem to have the same issue when https://carbondesignsystem.com/data-visualization/simple-charts is scanned with the accessibility checker.

### WCAG 2.1 Violation

IBM 1.1.1 Non-text Content

### Reproduction/example

https://codesandbox.io/p/sandbox/rc668d

### Steps to reproduce

Create a chart and scan it using the accessibility checker.  The sample provided is the donut chart, but any chart should produce the same violation.

![image](https://github.com/user-attachments/assets/c6a0e1b8-88b6-4100-becf-21e9b94b5d7e)

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by keithcha** [comment]
Similar to #17223 but different set of components.
